### PR TITLE
[DP][DDCE-1687] Fix accessibility issues (A)

### PR DESCRIPTION
**Pull request by DannyPennington97**
# DDCE-1687

~~Currently awaiting design and welsh sign-off but otherwise is ready~~
Has been signed off by both!

- Add timeout dialogue
- Change titles on csv upload pages
- Add labels for upload button

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -218,7 +218,6 @@ csv {
   uploadSizeLimit = 104857600
 }
 
-sessionTimeout {
-  timeoutSeconds = 900
-  time-out-countdown-seconds = 120
-}
+session {
+  timeout = 15.minutes
```
Comment (RafSobol) [inline conf/application.conf:222]: out of curiosity, how is that different to the previous version (bootstrap-play seems to allow both versions)? does this implicitly handle the time when the countdown appears?

Comment (DannyPennington97) [reply conf/application.conf:222]: Not gonna lie I don't think there's a difference, I saw the new one in their [guide](https://github.com/hmrc/play-frontend-hmrc#customisation) and figured we may as well use the one they mention. And yeah, the dialog appears at 2 minutes left by default (so if you set the timeout below that the dialog will always be there)

File: app/views/ers_main.scala.html [hunk 2]
```diff
@@ -34,6 +35,7 @@
     <meta name="format-detection" content="telephone=no"/>
     <meta http-equiv='Pragma' content='no-cache'/>
     <meta http-equiv='Cache-Control' content='no-cache, no-store, must-revalidate, private'/>
+    @hmrcTimeoutDialogHelper(signOutUrl = applicationConfig.signOut)
```
Comment (RafSobol) [inline app/views/ers_main.scala.html:38]: Axe finds a few accessibility issues with the timeout dialog, not sure if you had a look?

File: app/views/ers_main.scala.html [hunk 3]
```diff
@@ -58,7 +60,7 @@
     headBlock = Some(hmrcHead(Some(head))),
     headerBlock = Some(hmrcStandardHeader(
         serviceName = Some(messages("service.name")),
-        signOutUrl = None,
+        signOutUrl = Some(applicationConfig.signOut),
```
Comment (RafSobol) [inline app/views/ers_main.scala.html:63]: this is in DDCE-1912, would be good to link the tickets

Comment (RafSobol) [reply app/views/ers_main.scala.html:63]: (or leave it for then, your call)

Comment (DannyPennington97) [reply app/views/ers_main.scala.html:63]: Good spot, will do

**Pull request merged**